### PR TITLE
Pysbatch

### PR DESCRIPTION
**Pull request by luptior**
add pysbatch to condo-forge

### CODE REVIEW

File: recipes/pysbatch/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/luptior/pysbatch/archive/{{ version }}.zip
+  sha256: d07cce60a6072846d5783f42bd690644bccd2b739634bf850ba158ee9113d30e
+
+build:
+  number: 0
+  skip: True  # [py<35]
```
Comment (marcelotrevisani) [inline recipes/pysbatch/meta.yaml]: Please remove this selector and add ``noarch: python``

After that in ``host`` and also in ``run`` requirements put something like
```
- python >=3.5
```

File: recipes/pysbatch/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  description: |
+    Submit(sbatch) slurm cluster job inside python and avoid shell script for complicated pipeline jobs. 
+    For sbatch options, now only supports job name, memory size(in GBs), time limit(in days), dependency 
+    and ouput file. But you can use add_option parameter to add more.
+  doc_url: https://github.com/luptior/pysbatch/blob/master/README.md
+  dev_url: https://github.com/luptior/pysbatch
+
+extra:
+  recipe-maintainers:
+    - Luptior
```
Comment (marcelotrevisani) [inline recipes/pysbatch/meta.yaml]: I think your account in ``github`` is with the lower case ``L`` right?
So, I think it should be
```
- luptior
```

**Comment by marcelotrevisani** [comment]
LGTM!

Thanks to put this in ``conda-forge`` !

**Comment by luptior** [comment]
Thank you!

**Pull request merged**